devsetup/scripts/edpm-compute-bmaas.sh

```diff
@@ -0,0 +1,274 @@
+#!/bin/bash
+#
+# Copyright 2023 Red Hat Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+set -ex
+RHEL_IMAGE_URL=${RHEL_IMAGE_URL:-"https://images.rdoproject.org/centos9/master/rdo_trunk/current-tripleo/overcloud-hardened-uefi-full.qcow2"}
```

Comment:
Maybe we can remove something generic because we run this against both upstream/downstream and both rhel/centos based images will be used?

Reply:
RHEL is used everywhere, does not matter atm. https://github.com/openstack-k8s-operators/openstack-baremetal-operator/blob/master/api/v1beta1/openstackbaremetalset_types.go#L55

Reply:
ack